components/automate-ui/src/app/entities/environments/environment-details.selectors.ts

```diff
@@ -0,0 +1,28 @@
+import { createSelector, createFeatureSelector } from '@ngrx/store';
+import { EnvironmentDetailsEntityState, environmentDetailsEntityAdapter } from './environment-details.reducer';
+import { routeParams } from 'app/route.selectors';
+import { find } from 'lodash/fp';
```

Comment:
Convention is to put 3rd party in the first group, automate in the second group, and within each group from general to specific:
```suggestion
import { createSelector, createFeatureSelector } from '@ngrx/store';
import { find } from 'lodash/fp';

import { routeParams } from 'app/route.selectors';
import { EnvironmentDetailsEntityState, environmentDetailsEntityAdapter } from './environment-details.reducer';
```